### ISSUE TITLE
Scroll editor to page is not working but scroll pdf to cursor works fine.

### ISSUE THREAD

**Issue by BretG137** [jupyterlab/jupyterlab-latex#186]
## Description

Scroll editor to page is not working but scroll pdf to cursor works fine. 

## Reproduce

1. Go to '...'  PDF output 
2. Click on '...' right click 
3. Scroll down to '...'  and select "Scroll editor to page"
4. See error '...' does nothing

I searched github but found no one mention this issue.  Since everything else is working, including "scroll pdf" on the latex document, I don't see how it could be a configuration issue. Also, I have tried different browsers.

## Expected behavior

right clicking the pdf and selecting "scroll editor to page" should scroll the latex document to the location of the pdf file.

## Context

- Operating System and version:  debian 10
- Browser and version:  firefox 93 and chrome 90
- JupyterLab version:  3.0.0

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>
<details><summary>Command Line Output</summary>
$PATH:
        /home/user/code/jupyterenvironment/bin
        /home/user/.local/bin
        /home/user/.cargo/bin
        /usr/local/bin
        /usr/bin
        /bin
        /usr/local/games
        /usr/games

sys.path:
        /home/user/code/jupyterenvironment/bin
        /usr/lib/python37.zip
        /usr/lib/python3.7
        /usr/lib/python3.7/lib-dynload
        /home/user/code/jupyterenvironment/lib/python3.7/site-packages

sys.executable:
        /home/user/code/jupyterenvironment/bin/python

sys.version:
        3.7.3 (default, Jan 22 2021, 20:04:44) 
        [GCC 8.3.0]

platform.platform():
        Linux-4.19.0-18-amd64-x86_64-with-debian-10.11

which -a jupyter:
        /home/user/code/jupyterenvironment/bin/jupyter
        /home/user/.local/bin/jupyter

pip list:
        Package                   Version
        ------------------------- ---------
        absl-py                   0.12.0
        anyio                     3.1.0
        appdirs                   1.4.4
        argon2-cffi               20.1.0
        astroid                   2.6.6
        astunparse                1.6.3
        async-generator           1.10
        attrs                     21.2.0
        autopep8                  1.5.5
        Babel                     2.9.1
        backcall                  0.2.0
        bleach                    3.3.0
        cachetools                4.2.1
        cattrs                    1.8.0
        certifi                   2020.12.5
        cffi                      1.14.5
        chardet                   4.0.0
        cirq                      0.9.1
        colorama                  0.4.4
        cycler                    0.10.0
        Cython                    0.29.24
        decorator                 4.4.2
        defusedxml                0.7.1
        docstring-to-markdown     0.9
        entrypoints               0.3
        flake8                    3.9.2
        freezegun                 0.3.15
        future                    0.18.2
        gast                      0.3.3
        gitdb                     4.0.7
        GitPython                 3.1.17
        google-api-core           1.26.3
        google-auth               1.29.0
        google-auth-oauthlib      0.4.4
        google-pasta              0.2.0
        googleapis-common-protos  1.53.0
        graphviz                  0.16
        grpcio                    1.37.0
        h5py                      2.10.0
        idna                      2.10
        imageio                   2.9.0
        imageio-ffmpeg            0.4.3
        importlib-metadata        3.10.1
        ipykernel                 5.5.3
        ipython                   7.22.0
        ipython-genutils          0.2.0
        ipywidgets                7.6.3
        isort                     5.9.3
        jedi                      0.18.0
        Jinja2                    2.11.3
        json5                     0.9.5
        jsonschema                3.2.0
        jupyter                   1.0.0
        jupyter-client            6.1.12
        jupyter-console           6.4.0
        jupyter-core              4.7.1
        jupyter-drawio-export     0.8.0a1
        jupyter-lsp               1.5.0
        jupyter-resource-usage    0.5.1
        jupyter-server            1.8.0
        jupyter-server-mathjax    0.2.3
        jupyterlab                3.0.0
        jupyterlab-git            0.30.0
        jupyterlab-latex          3.1.0
        jupyterlab-lsp            3.9.1
        jupyterlab-pygments       0.1.2
        jupyterlab-server         2.6.0
        jupyterlab-system-monitor 0.8.0
        jupyterlab-topbar         0.6.1
        jupyterlab-widgets        1.0.0
        Keras-Preprocessing       1.1.2
        kiwisolver                1.3.1
        lazy-object-proxy         1.6.0
        llvmlite                  0.37.0
        lxml                      4.6.4
        Markdown                  3.3.4
        MarkupSafe                1.1.1
        matplotlib                3.4.1
        mccabe                    0.6.1
        mistune                   0.8.4
        moviepy                   1.0.3
        mpmath                    1.2.1
        nbclassic                 0.3.1
        nbclient                  0.5.3
        nbconvert                 6.0.7
        nbdime                    3.1.0
        nbformat                  5.1.3
        nest-asyncio              1.5.1
        networkx                  2.5.1
        notebook                  6.3.0
        numba                     0.54.1
        numpy                     1.18.5
        oauthlib                  3.1.0
        opt-einsum                3.3.0
        packaging                 20.9
        pandas                    1.2.4
        pandocfilters             1.4.3
        parso                     0.8.2
        pexpect                   4.8.0
        pickleshare               0.7.5
        Pillow                    8.2.0
        pip                       21.3.1
        pluggy                    1.0.0
        proglog                   0.1.9
        prometheus-client         0.10.1
        prompt-toolkit            3.0.18
        protobuf                  3.12.4
        psutil                    5.8.0
        ptyprocess                0.7.0
        pyasn1                    0.4.8
        pyasn1-modules            0.2.8
        pycodestyle               2.7.0
        pycparser                 2.20
        pydantic                  1.8.2
        pydocstyle                6.1.1
        pydot                     1.4.2
        pydotplus                 2.0.2
        pyflakes                  2.3.1
        pygls                     0.11.2
        Pygments                  2.8.1
        pylint                    2.9.6
        pyparsing                 2.4.7
        PyPDF2                    1.26.0
        pyrsistent                0.17.3
        python-dateutil           2.8.1
        python-jsonrpc-server     0.4.0
        python-language-server    0.36.2
        python-lsp-jsonrpc        1.0.0
        python-lsp-server         1.2.4
        pytz                      2021.1
        pyzmq                     22.0.3
        qtconsole                 5.0.3
        QtPy                      1.9.0
        requests                  2.25.1
        requests-cache            0.8.1
        requests-oauthlib         1.3.0
        rope                      0.21.0
        rsa                       4.7.2
        Rtree                     0.9.7
        scipy                     1.6.2
        seaborn                   0.11.1
        Send2Trash                1.5.0
        setuptools                58.2.0
        six                       1.15.0
        smmap                     4.0.0
        sniffio                   1.2.0
        snowballstemmer           2.1.0
        sortedcontainers          2.3.0
        sympy                     1.5
        tensorboard               2.5.0
        tensorboard-data-server   0.6.0
        tensorboard-plugin-wit    1.8.0
        tensorflow                2.3.1
        tensorflow-estimator      2.3.0
        tensorflow-quantum        0.4.0
        termcolor                 1.1.0
        terminado                 0.9.4
        testpath                  0.4.4
        tkintertable              1.3.3
        toml                      0.10.2
        tornado                   6.1
        tqdm                      4.60.0
        traitlets                 5.0.5
        typed-ast                 1.4.3
        typeguard                 2.13.0
        typing-extensions         3.7.4.3
        ujson                     4.2.0
        url-normalize             1.4.3
        urllib3                   1.26.4
        wcwidth                   0.2.5
        webencodings              0.5.1
        websocket-client          1.0.1
        Werkzeug                  1.0.1
        wheel                     0.37.0
        widgetsnbextension        3.5.1
        wrapt                     1.12.1
        yapf                      0.31.0
        zipp                      3.4.1

<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output</summary>

<pre>
Paste the output from your browser Javascript console here, if applicable.

</pre>
</details>

Thanks for all the hard work, everything works beautifully except for this one issue.

**Comment by Johan-Gronqvist** [comment]
I see the same symptom, and I also see an error message in the output of the terminal that started jupyterlab.

```
SyncTex command `synctex edit -o 6:0:0:/Workspace/PATH/TO/MY/PDF/File.pdf` errored with code: 255
```

When I run that command myself, I see the same error, but when I replace the first "/" by a "./", I see a presumably correct output.

My guess, only based on this, is that the command should include a relative path, but that it does not use the right form.
